units/ArmBots/armrock.lua

```diff
@@ -5,7 +5,7 @@ return {
 		buildcostenergy = 1000,
 		buildcostmetal = 120,
 		buildpic = "ARMROCK.DDS",
-		buildtime = 2015,
+		buildtime = 2010,
```

Comment:
2000? still under 1%

Reply:
missed this before, but this one in particular I didn't want to move much as it's a very staple earlygame unit - I want nice values, but definitely not at the expensive of broader balance. I restricted myself to 0.1% for these types of units.